Resources/Locale/en-US/prototypes/catalog/cargo/cargo-vending.ftl

```diff
@@ -31,9 +31,15 @@ ent-CrateVendingMachineRestockRobustSoftdrinks = { ent-CrateVendingMachineRestoc
 ent-CrateVendingMachineRestockSalvageEquipment = { ent-CrateVendingMachineRestockSalvageEquipmentFilled }
     .desc = { ent-CrateVendingMachineRestockSalvageEquipmentFilled.desc }
 
+ent-CrateVendingMachineRestockAstroVend = { ent-CrateVendingMachineRestockAstroVendFilled }
```

Comment:
lets keep all of our localization stuff separate, it makes it easier for translation and also avoids merge conflicts when we update.